### DISCUSSION TITLE
lolMiner 1.53 public beta 2

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1652] [General]
### Changes (beta 1 to beta 2)
- Significantly improved Kaspa performance in the Eth+Kas solver on RX 400/500 (approx. +120%), RX 5000 (approx. +20%) and  RX 6000 (approx. + 10%). 
- Improved Alephium performance in the Eth + Aleph solver on RX 5000 and RX 6000 GPUs
- Slightly reduced clock demand on Ethash only for RX 5000 GPUs
- Added experimental ETC+Kaspa solver for R9 285 / R9 380 (Tonga), RX Fury (Fiji) and RX 460 / 550 (Baffin) GPUs

### Fixes
- Fixed "conversion to type 'b'" error message in Kaspa stratum module
- Fixed miner displaying wrong version number and release month at startup
- Fixed miner setting up a dev fee connection for Kaspa - that was just unused, but occupied network resources

### Changes of beta 1 to 1.52a
- Added Eth/Etc/Ubiq + Kaspa dual mining for AMD RX400, RX500, RX5000 and RX 6000 series. Use --dualmode KASPADUAL to request this algorithm. Requires --dualpool to be set to a Kaspa stratum pool. GPUs that do not support this dual mode will automatically mine Eth/Etc/Ubiq only. This algorithms prefers the newer architectures. Fee is 1%+0% as for all dual mining modes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.53_beta_2'>lolMiner 1.53 public beta 2</a>.</em>

**Comment by OZILKER07** [comment]
sadece eth solo için lolminer kullanıyorum, hangi sürümü kullanmalıyım

i only use lolminer for eth solo, which version should i use

**Reply by djdeeles** [reply]
lolMiner 1.52a

**Comment by Tux0Topo** [comment]
Hi!! Can someone give any more data about "Slightly reduced clock demand on Ethash only for RX 5000 GPUs"??
